docs/troubleshooting-guide.md

```diff
@@ -15,10 +15,24 @@ Note that some of the screenshots might be outdated, but the instructions will r
 
 * **ERROR**: After downloading and installing Google Cloud SDK, running any `gcloud` command results in `gcloud: command not found` or alike.
 
-  **REASON**: You did not choose to update the `PATH` variable when installing the SDK.
+  **REASON**: Not update the `PATH` variable when installing the SDK.
```

Comment:
Phrasing seems a bit weird. How about "The PATH variable was not updated when installing the SDK"